src/fides/api/models/privacy_notice.py

```diff
@@ -456,18 +448,6 @@ class PrivacyNoticeHistory(PrivacyNoticeBase, Base):
         String, ForeignKey(PrivacyNotice.id_field_path), nullable=False
     )
 
-    def calculate_relevant_systems(self, db: Session) -> List[FidesKey]:
-        """Method to cache the relevant systems at the time to store on PrivacyPreferenceHistory for record keeping
-
-        A system is relevant if their data use is an exact match or a child of the notice's data use.
-        """
-        relevant_systems: List[FidesKey] = []
-        for system in db.query(System):
-            if self.applies_to_system(system):
-                relevant_systems.append(system.fides_key)
-                continue
-        return relevant_systems
-
```

Comment:
Obsolete functionality removed.  We no longer calculate "relevant systems" for the consent report upfront. This was removed for performance reasons to improve the speed of saving preferences, but in examining existing functionality, we believe it's not a necessary feature of the consent report.